aspnetcore/blazor/get-started.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Get started with Blazor
+author: guardrex
+description: Learn how to get started with Blazor.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/15/2019
+uid: blazor/get-started
+---
+# Get started with Blazor
+
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
+
+Get started with Blazor following either of the following experiences:
+
+* [Server-side Razor Components](#server-side-razor-components-experience)
+* [Client-side Blazor](#client-side-blazor-experience)
+
+## Server-side Razor Components experience
```

Comment:
Because *today* (if you want to merge to `live` today) this is the case. Soon, this won't be the case, and the upcoming changes on the next PR will resolve this. This is the best we can do if we take these changes to `live` today. I'm in a bit of *limbo* on the PR *today*.